contentcuration/contentcuration/frontend/accounts/components/MessageLayout.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+
+  <VApp>
+    <VLayout justify-center fill-height style="padding-top: 10%;">
+      <VFlex style="max-width: 900px;">
+        <h1 class="font-weight-bold headline text-xs-center">
+          {{ header }}
+        </h1>
+        <p class="text-xs-center subheading mt-1 mb-5">
+          {{ text }}
+        </p>
+        <div style="max-width: 400px; margin: 0 auto; text-align: center;">
+          <slot></slot>
+          <p class="mt-4">
+            <slot name="back">
+              <ActionLink :to="{ name: 'Main' }" :text="$tr('backToLogin')" />
+            </slot>
+          </p>
+        </div>
+      </VFlex>
+    </VLayout>
+  </VApp>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'MessageLayout',
+    props: {
+      header: {
+        type: String,
+        required: true,
+      },
+      text: {
+        type: String,
+        required: false,
+      },
+    },
+    $trs: {
+      backToLogin: 'Back to sign in',
+    },
+  };
+
+</script>
+
+
+<style lang="less" scoped>
+
+  .form-section {
+    font-size: 14px;
+  }
```

Comment:
I see `.form-section` being repeated in many components in a `scoped` style block, but the class is never actually specified in a template.

Is this "reaching in" to the implementation of some vuetify component? If so, there's probably a less brittle and more DRY way of achieving this. Happy to consult on slack.

Reply:
Ah nope, it was me just not removing it in the original file and copying that file everywhere haha. Removed!